pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -704,7 +742,7 @@ func (r *ClickHouseReader) GetServiceOverview(ctx context.Context, queryParams *
 			count(*) as numErrors
 		FROM %s.%s
 		WHERE serviceName = @serviceName AND name In @names AND timestamp>= @start AND timestamp<= @end AND statusCode=2`,
-		r.TraceDB, r.indexTable,
+		r.TraceDB, r.traceTableName,
```

Comment:
Shouldn't there be a bucket filter for `GetServiceOverview ` queries as well?

Reply:
have added the filter, but do you know from where `GetServiceOverview` is called. The services page and the detail of a signle service doesn't call this API

Reply:
I am not sure. I would suggest talk to someone who knows what APIs are used on frontend and remove the ones that are no longer used.

Reply:
Remoing this in another PR.